lib/main.coffee

```diff
@@ -12,26 +12,24 @@ module.exports =
       description: 'Command or path to executable. Use %p for current project directory (no trailing /).'
     pythonPath:
       type: 'string'
-      default: ''
-      description: 'Paths to be added to $PYTHONPATH. Use %p for current project directory or %f for the directory of
-        the current file.'
+      default: '%(PYTHONPATH), %f, %p'
+      description: 'Comma delimited list of paths to set $PYTHONPATH to. Use %p for current project directory,
+        %f for the directory of the current file, or %(VAR_NAME) to use a pre-existing environment variable.'
     rcFile:
       type: 'string'
       default: ''
-      description: 'Path to pylintrc file. Use %p for the current project directory or %f for the directory of the
-        current file.'
+      description: 'Path to pylintrc file. Use %p for current project directory, %f for the directory of the
+        current file, or %(VAR_NAME) to use a pre-existing environment variable.'
     workingDirectory:
       type: 'string'
       default: '%p'
-      description: 'Directory pylint is run from. Use %p for the current project directory or %f for the directory
-        of the current file.'
+      description: 'Directory pylint is run from. Use %p for current project directory, %f for the directory of
+        the current file, or %(VAR_NAME) to use a pre-existing environment variable.'
     messageFormat:
       type: 'string'
       default: '%i %m'
-      description:
-        'Format for Pylint messages where %m is the message, %i is the
-        numeric mesasge ID (e.g. W0613) and %s is the human-readable
-        message ID (e.g. unused-argument).'
+      description: 'Format for Pylint messages where %m is the message, %i is the numeric mesasge ID
```

Comment:
Any reason you pulled this up into two lines? Shorter lines are _much_ better if you run more than one editor side by side.